packages/ckeditor5-emoji/src/emojimention.ts

```diff
@@ -81,9 +81,21 @@ export default class EmojiMention extends Plugin {
 		this._emojiDropdownLimit = editor.config.get( 'emoji.dropdownLimit' )!;
 		this._skinTone = editor.config.get( 'emoji.skinTone' )!;
 
-		const mentionFeedsConfigs = editor.config.get( 'mention.feeds' )! as Array<MentionFeed>;
+		this._setupMentionConfiguration( editor );
+	}
+
+	/**
+	 * Initializes the configuration for emojis in the mention feature.
+	 * If the marker used by emoji mention is already registered, it displays a warning.
+	 * If emoji mention configuration is detected, it does not register it for a second time.
+	 */
+	private _setupMentionConfiguration( editor: Editor ) {
+		const mentionFeedsConfigs = editor.config.get( 'mention.feeds' )! as Array<EmojiMentionFeed>;
```

Comment:
I'm not sure if I'm supposed to use `editor.config.get( 'mention.feeds' )` and `editor.config.get( 'mergeFields.prefix' )!` outside of a constructor method. I'm doing it, because it's hard to meet the coverage without it, and also I have seen some examples in the ckeditor5 code that do the same.

Reply:
It's fine :+1: